frontend/src/App.js

```diff
@@ -1,21 +1,26 @@
-import React from 'react';
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import NavBar from "./components/NavBar";
+import Carousel from "./components/Carousel";
+import SignIn from "./components/SignIn";
 import Form from "./components/Form";
-import GameChoice from "./components/GameChoice";
-import GameList from "./GameList";
-
-
+import NewsFeed from "./components/NewsFeed";
 
 function App() {
   return (
     <div className="App">
-      <div className="bigGamePage">
-      <div className="gamePage">
-      {GameList.map ((game) => {
-      return (<GameChoice name={game.name} picture={game.picture} />)})}
-      </div>
-      </div>
-    </div>)
-};
-
+      <Switch>
```

Comment:
I think there was a merge problem here.

Reply:
Ok. I 've deleted the first switch and added the Navbar to App.js